lua/telescope/_extensions/project_actions.lua

```diff
@@ -63,8 +63,12 @@ project_actions.delete_project = function(prompt_bufnr)
   require 'telescope'.extensions.project.project()
 end
 
-project_actions.find_project_files = function(prompt_bufnr)
-  builtin.find_files({cwd = actions.get_selected_entry(prompt_bufnr).value})
+project_actions.find_project_files = function(prompt_bufnr, change_dir)
+  local dir = actions.get_selected_entry(prompt_bufnr).value
+  if change_dir then
+    vim.cmd("cd "..dir)
```

Comment:
`vim.fn.execute("cd " .. dir, "silent")` might be better. unless its important to print the new dir to command line

Reply:
Changed, thanks!